docs/src/_getting-started/index.md

```diff
@@ -908,6 +908,7 @@ schedules:
 Once your raw data has arrived in your data warehouse, its schema will likely need to be transformed to be more appropriate for analysis.
 
 To help you achieve this, Meltano supports transformation using [`dbt`](https://www.getdbt.com/).
+If you already have an existing dbt project that you'd like to migrate to Meltano, check out the [existing dbt project guide](https://docs.meltano.com/guide/existing-dbt-project) for more details.
```

Comment:
Meta question - is this more appropriate for a Tutorial? I'm wishing we didn't have it in the side bar along with the other Guides, but I understand the logic of putting it there. 

@sbalnojan do you have stronger opinions on this?

Reply:
@tayloramurphy thanks for the suggestions!

In terms of guide vs tutorial I was debating myself but went with guide since I dont actually show an example or anything. I felt like a tutorial would be more like if I showed how to migrate the jaffle shop project to Meltano, thats wroth while too but I didnt have the time to do it. No strong feelings either way, I'd be fine swapping it to tutorial if you prefer.

Reply:
@pnadolny13 how about we keep it in the guide but put it under the "Advanced Topics" section?

Reply:
@pnadolny13 / @tayloramurphy I would consider this a (diataxis) guide, as Pat explained. Right now I won't mind much where to put it because the sections are quite messy anyways. My current plan is to create a new "How To" (parallel to the guides section) section and slowly migrate all the "actual guides" into it.

Reply:
@sbalnojan great - thanks for the guidance on that.

@pnadolny13 let's rename the guide to "Migrate and Existing dbt Project" and slide it down in the overall Guide section. Then I think we'll be good to go.

Reply:
@tayloramurphy sounds good - I updated it